### PR TITLE
Import missing translations

### PR DESCRIPTION
**Pull request by 1pretz1**
These languages were originally missed out from the supplier (ASP) and so this PR imports them.

Original import PR: https://github.com/alphagov/government-frontend/pull/2285

More info in the commits ->

Trello:
https://trello.com/c/Xz78vNR4/2757-import-newly-sourced-translations-into-locale-files-5

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: config/locales/gu.yml [hunk 1]
```diff
@@ -371,10 +371,10 @@ gu:
     release_date: રીલીઝ તારીખ
   travel_advice:
     alert_status:
-      avoid_all_but_essential_travel_to_parts: <abbr title="Foreign and Commonwealth Office">વિદેશી - કોમનવેલ્થ અને વિકાસ કચેરી</abbr> જરૂરી મુસાફરી સિવાય દેશના કેટલાક ભાગોમાં પ્રવાસ ન કરવાની સલાહ આપે છે.
-      avoid_all_but_essential_travel_to_whole_country: <abbr title="Foreign and Commonwealth Office">વિદેશી - કોમનવેલ્થ અને વિકાસ કચેરી</abbr> જરૂરી મુસાફરી સિવાય, સંપૂર્ણ દેશમાં પ્રવાસ ન કરવાની સલાહ આપે છે.
-      avoid_all_travel_to_parts: <abbr title="Foreign and Commonwealth Office">વિદેશી - કોમનવેલ્થ અને વિકાસ કચેરી</abbr> દેશના કેટલાક ભાગોમાં કોઈ પણ પ્રકારનો પ્રવાસ ન કરવાની સલાહ આપે છે.
-      avoid_all_travel_to_whole_country: <abbr title="Foreign and Commonwealth Office">વિદેશી - કોમનવેલ્થ અને વિકાસ કચેરી</abbr> સંપૂર્ણ દેશમાં કોઈ પણ પ્રકારનો પ્રવાસ ન કરવાની સલાહ આપે છે.
+      avoid_all_but_essential_travel_to_parts_html: <abbr title="Foreign and Commonwealth Office">વિદેશી - કોમનવેલ્થ અને વિકાસ કચેરી</abbr> જરૂરી મુસાફરી સિવાય દેશના કેટલાક ભાગોમાં પ્રવાસ ન કરવાની સલાહ આપે છે.
```
Comment (ChrisBAshton) [inline config/locales/gu.yml:468]: Didn't know that `_html` was a feature! Could you amend the commit message to link to the documentation for it? https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

File: config/locales/ur.yml [hunk 2]
```diff
@@ -371,10 +371,10 @@ ur:
     release_date: اجراء کی تاریخ
```
Comment (ChrisBAshton) [inline config/locales/ur.yml:465]: I noticed [two keys](https://github.com/alphagov/government-frontend/blob/d2789383a9117c7e54d7d18834fc945c7a1bee19/config/locales/ur.yml#L38-L39) whose values contain HTML, but the key names don't end in `_html` - were they missed?

File: config/locales/ar.yml [hunk 3]
```diff
@@ -631,6 +631,7 @@ ar:
     engage_with_gov:
     fcdo_bloggers:
     follow:
+    follow_links:
```
Comment (ChrisBAshton) [inline config/locales/ar.yml:634]: Do you know what the next steps are in terms of getting this key translated?

**Review by ChrisBAshton — COMMENTED** [review]
This is looking good - hardly any comments from me! Just a couple of questions below:

File: config/locales/ur.yml (hunk 2 above)
Comment (1pretz1) [reply config/locales/ur.yml:465]: This confused me for a second! 😆 Those keys are actually unused and have actually been removed in the the following commit: https://github.com/alphagov/government-frontend/pull/2305/commits/45d2065a38d14f8bad40409fe7903613b51deb3b#diff-f8ecf4cd17cdd1ecc6f81401e44eba0a0e3c55ff9cd30b5acd1a3c4029fc5638L38-L39

File: config/locales/gu.yml (hunk 1 above)
Comment (1pretz1) [reply config/locales/gu.yml:468]: Sure 👍

File: config/locales/ar.yml (hunk 3 above)
Comment (1pretz1) [reply config/locales/ar.yml:634]: Nope, unfortunately! Currently there is no contingency plan for importing translations. The mission got cut short 😢  I'm hoping the next team that inherits this work (Find and view which I'm moving to) can prioritise it, but we'll see.

File: config/locales/ur.yml (hunk 2 above)
Comment (ChrisBAshton) [reply config/locales/ur.yml:465]: Great - thanks for confirming!

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍 thanks for answering my questions. ⭐

**Pull request merged**